lib/vsc/install/shared_setup.py

```diff
@@ -304,6 +307,33 @@ def reload_vsc_modules(self):
         for name in reload_vsc_modules:
             __import__(name)
 
+    def force_xmlrunner(self):
+        """
+        A monkey-patch attempt to run the tests with
+        xmlrunner.XMLTestRunner(output=xyz).run(suite)
+
+        E.g. in case of jenkins and you want junit compatible reports
+        """
+        import xmlrunner
+        import unittest
+
+        xmlrunner_output = self.test_xmlrunner
+
+        class OutputXMLTestRunner(xmlrunner.XMLTestRunner):
+            """Force the output"""
+            def __init__(self, *args, **kwargs):
+                kwargs['output'] = xmlrunner_output
+                xmlrunner.XMLTestRunner.__init__(self,*args, **kwargs)
+
+        main_orig = unittest.main
```

Comment:
`unittest_main` is just a reference to the original `unittest.main`, I don't see a difference with `main_orig` (other than the name ;-))